R/make_plots.R

```diff
@@ -5,6 +5,9 @@ library(scales)
 library(reshape2)
 library(stringi)
 
+# define standard colors
+colorvec <- c('steelblue', 'darkolivegreen4', 'lightgoldenrod2', 'sienna2', 'tomato3')
```

Comment:
I think this a great idea. But maybe we should start to align a bit more on a color palette across all our tools?

In our javascript tools, think we are currently using two basic schemes:

metricsgraphics base:
- #4040e8 (residential demand blue)
- #05b378 (residential goal green)

bootstrap base:
- #3EAB45 (green)
- #B9D14C (chartreuse)
- #D9C24F (yellow)
- #D99F4F (orange)
- #D9534F (red)
and think the blue that goes with the bootstrap base would be #39668C.

Any opposition to using one of these?

Reply:
Standards across tools seems reasonable to me